lockfile_compat.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 // Deprecated: Use lockfile.*LockFile.
-type Locker = lockfile.Locker //lint:ignore SA1019 // lockfile.Locker is deprecated
+type Locker = lockfile.Locker //nolint:staticcheck // SA1019 ignore this
```

Comment:
“ignore this” is not a useful comment. At least say what the warning is about, without users having to look it up; or ideally, say why ignoring the warning is fine.

(Just a quick note, I’ll check the other changes later.)